build/ci.bazelrc

```diff
@@ -43,7 +43,7 @@ build:ci-test --config=v8-codegen-opt
 # LLVM produces a bit more debug info on macOS by default to facilitate debugging with
 # LLDB. This is not needed for this use case, so disable it using -fno-standalone-debug –
 # this is already the default for Linux/Windows.
-build:ci-limit-storage --build_tag_filters=-off-by-default,-benchmark
+build:ci-limit-storage --build_tag_filters=-off-by-default,+workerd-benchmark
```

Comment:
Tag filters mean that "Only those targets will be built that contain at least one included tag and do not contain any excluded tags." so this change would change the configuration to exclusively run benchmark tests, which is not what we want. Just having `-off-by-default,-workerd-benchmark` would keep the same behavior as before right?

Reply:
You're right. Reverted this.